src/platform/user/profile/vet360/components/AddressField/AddressEditModal.jsx

```diff
@@ -21,6 +22,7 @@ export default class AddressEditModal extends React.Component {
   };
 
   getInitialFormValues = () =>
+    this.props.modalData ||
```

Comment:
This is the key: fill the form with modalData if it exists and then fall back to using the data that's stored in VA Profile